ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+
+- name: install Rust deps
+  apt:
+    name:
+      - pkg-config
+      - build-essential
+      - libssl-dev
+    state: present
+
+- name: install rustup in userspace for root
+  shell: |
+    set -eu
+    RUSTUP_VERSION=1.24.3
+    RUSTUP_SHA="a3cb081f88a6789d104518b30d4aa410009cd08c3822a1226991d6cf0442a0f8"
+    curl --proto '=https' --tlsv1.2 -sSf -O \
+      https://raw.githubusercontent.com/rust-lang/rustup/${RUSTUP_VERSION}/rustup-init.sh
+    echo "${RUSTUP_SHA}  rustup-init.sh" | sha256sum --check --
+    sh rustup-init.sh --default-toolchain nightly -y --component rust-src
+
+- name: Configure update script service
+  copy:
+    src: team_login/
+    dest: "{{ vars_team_login_path }}"
+
+- name: Build team login cron job
+  shell: "cd {{ vars_team_login_path }} && PATH=$PATH:$HOME/.cargo/bin cargo build"
+
+- name: Enable message of the day
+  template:
+    src: motd_sshd_config
+    dest: /etc/ssh/sshd_config.d/motd
+
+- name: Disable automatically generated message of the day
+  shell: "sed -i -e 's/^session    optional     pam_motd/#session    optional     pam_motd/' /etc/pam.d/sshd"
+
+- name: Set up message of the day
+  template:
+    src: motd_rules
+    dest: /etc/motd
+
+- name: Install the team login binary
+  shell: "cp {{ vars_team_login_path }}/target/debug/team_login /etc/cron.team_login"
+
+- name: Set up the team login cron job
```

Comment:
Not a blocker for the initial prototyping, but you mentioning the sync is currently failing due to the team repo changes not being applied reminded me that we don't have proper monitoring in place for cronjobs failing, but we do have monitoring in place for systemd units failing.

Could the cronjob be switched to a systemd timer/service combo? You can see an example in this repo, with the corresponding [.service](https://github.com/rust-lang/simpleinfra/blob/master/ansible/roles/docker/templates/update-images/docker-images-update.service) and [.timer](https://github.com/rust-lang/simpleinfra/blob/master/ansible/roles/docker/templates/update-images/docker-images-update.timer).